src/Directory.Build.props

```diff
@@ -16,7 +16,7 @@
     <NetRuntime>net6.0</NetRuntime>
     <XunitVersion>2.4.2</XunitVersion>
     <TestSdkVersion>17.3.0</TestSdkVersion>
-    <AkkaVersion>1.4.43</AkkaVersion>
```

Comment:
To simulate the upgrade - need to start this solution with v1.4 Akka.NET, _create data here_, AND THEN CHANGE TO V1.5. If you go straight to v1.5 without populating data, nothing will happen.